### PR TITLE
Created conservatory

### LINKED ISSUE #151: Create conservatory room
## Expected Behaviour

Create  conservatory room ( + assets needed)
Add new room between library & dining room for level 4
Closed door to the conservatory in level 1 & 3
Server room door moved to bottom in level 2 

## Actual Behaviour

No conservatory at the moment

## Steps to Reproduce Behaviour

1. Play the blooming game

### PR DESCRIPTION
**Pull request by Alice-Jaffray**
**Description of work.**

Create  conservatory room ( + assets needed)
Add new room between library & dining room for level 4
Closed door to the conservatory in level 1 & 3
Server room door moved to bottom in level 2 

**To test:**

check it out! FYI the minimap will go in with the minimap ticket 

Fixes #151

### CODE REVIEW

**Review by cailafinn — CHANGES_REQUESTED** [review]
Looks good, just a few notes:
- Objects need to be on the Character sorting layer or Sherry will appear in front of them
- There is no hitbox for the edge of the room, so you can just walk out the top, bottom, or sides 
- The question in the conservatory already exists in the server room, attached to the desk. I would also prefer if we used a different question in the conservatory as the password one I think makes sense being attached to the server room PC. The server tower question might be a better one to move over

**Comment by Alice-Jaffray** [comment]
Lol completely forgot to put an edge collider and delete the question from the server room

**Review by cailafinn — CHANGES_REQUESTED** [review]
The desk still has the interactable tag and the object interaction script on it. 

I would also prefer if the password question stayed on the desk object and one of the other questions was moved to the conservatory that doesn't have a good relation to the object it's attached to

**Review by cailafinn — CHANGES_REQUESTED** [review]
Almost perfect, but the question in the conservatry has the incorrect "wrong answer" text on it. just needs changing to the one from the hall flower pot question

File: project/sherLOCKED/Assets/Rooms/Level4/Conservatory.unity [hunk 1]
```diff
@@ -300,8 +300,8 @@ MonoBehaviour:
   correctAnswer: {fileID: 413908942}
   extraInfo: {fileID: 413908939}
   questionUI: {fileID: 413908943}
-  wrongText: '"Leetspeak" isn''t very secure any more and random strings of letters
-    aren''t very memorable. '
+  wrongText: 'Whoops. It''s about making security as easy as possible for the user.
+    Whoops. It''s about making security as easy as possible for the user. '
```
Comment (ConorMFinn) [inline project/sherLOCKED/Assets/Rooms/Level4/Conservatory.unity:304]: This appears twice when you get the question wrong.

Comment (cailafinn) [inline project/sherLOCKED/Assets/Rooms/Level4/Conservatory.unity]: This appears twice if you get the question wrong.

**Review by cailafinn — CHANGES_REQUESTED** [review]

**Review by cailafinn — APPROVED** [review]

**Pull request merged**